deepchecks/nlp/checks/data_integrity/property_label_correlation.py

```diff
@@ -27,7 +27,7 @@
 
 PLC = t.TypeVar('PLC', bound='PropertyLabelCorrelation')
 
-pps_url = 'https://docs.deepchecks.com/en/stable/checks_gallery/tabular/' \
+pps_url = 'https://docs.deepchecks.com/en/stable/tabular/checks_gallery/' \
```

Comment:
this link doesn't work.
Didn't check the others, please verify that they work

Reply:
They will work, after we will merge